pyproject.toml

```
@@ -7,6 +7,7 @@ build-backend = 'maturin'
 
 [project]
 name = 'pydantic_core'
+description = "core functionality for Pydantic validation and serialization."
```

It looks like the convention is to use capital letter at the start and no period, so I'll apply that.

```suggestion
description = "Core functionality for Pydantic validation and serialization"
```